source/products/roland/haus-und-grund/webapp.rst

```diff
@@ -0,0 +1,59 @@
+Webapp
+======
+
+Example Kasko JS embed code
+---------------------------
+
+.. code:: html
+
+    <script type="text/javascript" src="https://eu1.kaskojs.com/v2"></script>
+    <div id="kasko-widget-container"></div>
+    <script>
+      Kasko.Setup({
+        container: "kasko-widget-container",
+        touchpoint: "tp_14a00f394fe37d85348bce9908907",
+        mode: "fullscreen",
+        language: 'de',
+        metadata: {
+          tan: response.tan,
```

Comment:
Where this comes from?

Reply:
its prefilled from clients, just in a separate parameter `?metadata={}`